solutions/cses-1081.mdx

```diff
@@ -145,3 +147,24 @@ int main() {
 	}
 }
 ```
+</CPPSection>
+<PySection>
+```py
+MAXX = int(1e6 + 5)
+n = int(input())
+arr = list(map(int, input().split()))
+p = [0 for i in range(MAXX)]
+
+for i in range(n) :
+	p[arr[i]] += 1
```

Comment:
looks good, did you mean to put spaces before the colons?

Reply:
yes that is how i write it if it is blasphemy i will gladly get burned at the stake